src/libraries/System.Private.CoreLib/src/System/AppContext.AnyOS.cs

```diff
@@ -30,5 +33,51 @@ private static string GetBaseDirectoryCore()
 
             return directory;
         }
+
+        internal static void LogSwitchValues(RuntimeEventSource ev)
+        {
+            if (s_switches is not null)
+            {
+                lock (s_switches)
+                {
+                    foreach (var (k, v) in s_switches)
+                    {
+                        // Convert bool to int because it's cheaper to log (no boxing)
+                        ev.LogAppContextSwitch(k, v ? 1 : 0);
```

Comment:
Nit:
```suggestion
                    foreach (KeyValuePair<string, bool> kvp in s_switches)
                    {
                        // Convert bool to int because it's cheaper to log (no boxing)
                        ev.LogAppContextSwitch(kvp.Key, kvp.Value ? 1 : 0);
```